python_practice/python_practice.md

```diff
@@ -0,0 +1,137 @@
+comment: Use the basics of Python coding, data transformation, and data visualization to work with real data. 
+long_description: When learning Python for data science, the ultimate goal is to be able to put all of the pieces together to analyze a dataset. This module aims to provide a data science task in order to help learners practice Python skills in a real-world context. 
+estimated_time: 1 hour
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+- Import a dataset from an online database
+- Recode data and change variable types in a dataframe
+- Use exploratory data visualization to identify trends in data and generate hypotheses
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Python Practice
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?:** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites:** Learners should be familiar with [the basics of Python coding](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/python_basics_writing_python_code/python_basics_writing_python_code.md), including [data transformation with pandas](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/pandas_transform/pandas_transform.md) and [data visualization with matplotlib and seaborn](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/data_visualization_in_seaborn/data_visualization_in_seaborn.md). Learners should also have access to Python, either on their own computer or in the cloud. 
+
+**Learning Objectives**:
+
+@learning_objectives
+
+</div>
+
+## Lesson Preparation
+
+For this module, you'll need access to Python, either on your own computer or in the cloud. For details about how to download Python or use Google Colab (a cloud-based notebook environment), you can take a look at the ["Accessing Python" section of our Demystifying Python module](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_python/demystifying_python.md#9). 
+
+## The Data
+
+For this practice module we will be using real, publicly-available cervical cancer data from [UC Irvine's machine learning data repository](http://archive.ics.uci.edu/ml/index.php); you can download those data here: [http://archive.ics.uci.edu/ml/machine-learning-databases/00383/risk_factors_cervical_cancer.csv](http://archive.ics.uci.edu/ml/machine-learning-databases/00383/risk_factors_cervical_cancer.csv). If you have done the R Practice Module (**link required**), this dataset might be familiar! 
+
+## Topics to Review
+
+- Importing Python libraries 
+- Reading a CSV and creating a data frame
+- Editing/recoding data
+- Exploratory data visualization
+
+Stretch topics
```

Comment:
What does" stretch topics" mean?

Reply:
I'm using it to mean topics that we haven't explicitly taught in a module, but that learners should have all of the skills to use with a little searching on their own. I can see if I can make it clearer.

Reply:
Hopefully this is a bit clearer now!